test.sh

```
@@ -0,0 +1,385 @@
+  "OS     : \(.Os // "n/a")"
+' "$TMP_WORKDIR/skopeo.json"
+note "Relevant labels:"
+jq -r '
+  .Labels // {}
+  | to_entries[]
+  | select(.key | test("name|version|release|summary|vendor|source"; "i"))
+  | "  \(.key)=\(.value)"
+' "$TMP_WORKDIR/skopeo.json" | sort || true
+
+section "Runtime distribution evidence"
+container_py <<PY > "$TMP_WORKDIR/runtime.json"
```

Shellcheck (0.11.0)

[info] Use container_py "$@" if function's $1 should mean script's $1.

(SC2119)